scripts/ci/test_index.py

```diff
@@ -16,7 +16,7 @@
 import hashlib
 import shutil
 
-from distutils.version import LooseVersion
+from packaging import version
```

Comment:
`distutils.version.LooseVersion` does not support PEP440 which we follow as discussed in https://stackoverflow.com/a/11887885/2199657